audio/libzita-resampler/Portfile

```
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                libzita-resampler
+version             1.8.0
+revision            0
+
+categories          audio
+license             GPL-3+
+maintainers         {@aeiouaeiouaeiouaeiouaeiouaeiou outlook.com:aeioudev} openmaintainer
+
+description         Libzita-resampler is a C++ library for resampling audio \
+                    signals.
+long_description    ${description} It is designed to be used within a \
```

If reusing the `description` variable in `long_description`, you need to use the Tcl expansion operator (`{*}`):

```suggestion
long_description    {*}${description} It is designed to be used within a \
```